### DISCUSSION TITLE
listContacts call with SESV2 broken for .NET and JAVA?

### DISCUSSION THREAD

**Discussion by Aracif** [aws/aws-sdk-java#2815] [Q&A] [answered]
It's been days now and I've gotten 0 help from Amazon. I opened a ticket a few days ago and it mysteriously disappeared. Please see this stackoverflow question for details that i posted several days ago [https://stackoverflow.com/questions/73059306/how-to-fix-aws-403-error-check-your-aws-secret-access-key-and-signing-method-w.  ](url)

It seems that the listContacts functionality for all languages of the V2 SDK Apis is broken. I've tried c#, java personally and found this issue referenced somewhere else on stackoverflow unanswered. I've spent over a week wasted now. Does anyone, anywhere know what the deal is with this? Without this functionality it renders the entire V2 SES api useless since I can't get any of my contacts. Is there some alternative method to get my list of contacts programmatically? Also, why would the bug issue I created be delete with no trace?

**Comment by debora-ito** [comment, marked as answer]
Hi @Aracif your issue did not disappear, it got transferred to the central `aws/aws-sdk` repository here: https://github.com/aws/aws-sdk/issues/307.

We keep issues in that repository that have impact across all the SDKs.

Regarding the error, the SES team is aware of the issue. As far as I know there's no workaround using the Java SDK as the issue is on the server side. We are constantly asking for a fix, and will post any updates in the tracking github issue.

**Comment by Aracif** [comment]
Got it , sorry about that I was not aware. Thanks very much for the info, much appreciated.
